### PR TITLE
Array shift

### PR DESCRIPTION
**Pull request by rnmessick**
- [x ] Top-level README “Table of Contents” is updated
 - [x ] Feature tasks for this challenge are completed
 - [x ] Unit tests written and passing
     - [x ] “Happy Path” - Expected outcome
     - [ x] Expected failure
     - [x ] Edge Case (if applicable/obvious)
 - [ x] README for this challenge is complete
     - [x ] Summary, Description, Approach & Efficiency, Solution
     - [x ] Link to code
     - [x ] Picture of whiteboard

### CODE REVIEW

File: code401challenges/src/main/java/code401challenges/ArrayShift.java [hunk 1]
```diff
@@ -2,6 +2,19 @@
 
 public class ArrayShift {
     public static int[] arrayShift(int[] arrToBeShifted, int intToBeAdded) {
+        int[] shiftedArr = new int[arrToBeShifted.length + 1];
+        int middleIdx = (int) Math.ceil(arrToBeShifted.length / 2.0); // turns out this must be a decimal
```
Comment (evanslaton) [inline code401challenges/src/main/java/code401challenges/ArrayShift.java]: Yes, if you divide two `int`s in java it will give you an `int`!

File: code401challenges/src/test/java/code401challenges/ArrayShiftTest.java [hunk 2]
```diff
@@ -7,6 +7,29 @@
...
+    @Test
+    public void testArrayShift_fail() {
+        assertFalse("Insert should be at middle not end", new int[]{1, 2, 3, 4, 5}, ArrayShift.arrayShift(new int[]{1, 2, 3, 4}, 5));
+        assertFalse("Insert should be at middle not beginning", new int[]{5, 1, 2, 3, 4}, ArrayShift.arrayShift(new int[]{1, 2, 3, 4}, 5));
+        assertFalse("Insert should not be before middle index on odd length arrays", new int[]{1, 2, 5, 3, 4, 5}, ArrayShift.arrayShift(new int[]{1, 2, 3, 4, 5}, 5));
+
+
+    }
+
+    private void assertFalse(String s, int[] ints, int[] arrayShift) {
```
Comment (evanslaton) [inline code401challenges/src/test/java/code401challenges/ArrayShiftTest.java]: You definitely want to delete unused method declarations!
